pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: instabug_http_client
 description:
   This package is an add on to instabug_flutter. It intercepts any requests performed
   with http Package and sends them to the report that will be sent to the dashboard.
-version: 2.3.0
+version: 2.4.0
```

Comment:
@dustin-graham I think we should revert this change, this will be done once we release the package.

Reply:
@a7medev Sorry for the slow respomse. Now that the base package is 2.4.0, perhaps this is now OK?